qasp/agile_bpa_qasp.md

```diff
@@ -0,0 +1,106 @@
+PERFORMANCE BASED QUALITY ASSURANCE SURVEILLANCE PLAN (QASP)
+=============================================================
+
+{{ project_name }}
```

Comment:
On some of our templates we are using camelcase, and on others underscore. I think we should switch to one or the other.